src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Standard/NativeMethods.cs

```diff
@@ -2632,11 +2632,12 @@ public static HRESULT DwmSetWindowAttributeSystemBackdropType(IntPtr hwnd, DWMSB
         }
 
         [SuppressMessage("Microsoft.Performance", "CA1811:AvoidUncalledPrivateCode")]
-        public static HRESULT DwmSetWindowAttributeUseImmersiveDarkMode(IntPtr hwnd, bool useImmersiveDarkMode)
+        public static bool DwmSetWindowAttributeUseImmersiveDarkMode(IntPtr hwnd, bool useImmersiveDarkMode)
         {
             Assert.IsTrue(Utility.IsWindows11_22H2OrNewer);
             var pvAttribute = useImmersiveDarkMode ? 0x1 : 0x0;
-            return _DwmSetWindowAttribute(hwnd, DWMWA.USE_IMMERSIVE_DARK_MODE, ref pvAttribute, Marshal.SizeOf(typeof(int)));
+            var dwmResult = _DwmSetWindowAttribute(hwnd, DWMWA.USE_IMMERSIVE_DARK_MODE, ref pvAttribute, Marshal.SizeOf(typeof(int)));
```

Comment:
I've originally replaced this one in #9220, however could we use `sizeof(int)` here?

Reply:
For now rest of the files have Marshal.SizeOf, so I am leaving this as it is. When we take your PR, we will do this modification along with it.

Reply:
I understand, thanks.